Elevel/Application/Features/QuestionCommands/GetQuestionListQuery.cs

```diff
@@ -0,0 +1,78 @@
+﻿using AutoMapper;
+using Elevel.Application.Interfaces;
+using Elevel.Application.Pagination;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Elevel.Application.Features.QuestionCommands
+{
+    public class GetQuestionListQuery
+    {
+        public class Request : PagedQueryBase, IRequest<Response>
+        {
+            public Guid Id { get; set; }
```

Comment:
In this case we don't need all of these fields because when we request all question we only need PagedQueryBase.

Reply:
Deleted except for Id

Reply:
For which purpose you have Id for GetList request? how you want to use it? Just get one question? does it have any sense if you can get 1 question by GetDetailById query>?